src/main/resources/function/listFunctions.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied. See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+<template xmlns="http://ws.apache.org/ns/synapse" name="listFunctions">
+    <parameter name="functionVersionToBeListed"
+               description="Set to ALL to include entries for all published versions of each function."/>
+    <parameter name="marker"
+               description="Specify the pagination token returned by a previous request to retrieve the next page of results. "/>
+    <parameter name="masterRegion" description="The region of the master function."/>
+    <parameter name="apiVersionListFunctions" description="API version used for ListFunctions method."/>
+    <parameter name="maxItems" description="Number of functions to get in the response."/>
+    <sequence>
+        <property name="uri.var.functionVersion" expression="$func:functionVersionToBeListed"/>
+        <property name="uri.var.marker" expression="$func:marker"/>
+        <property name="uri.var.masterRegion" expression="$func:masterRegion"/>
+        <property name="uri.var.apiVersion" expression="$func:apiVersionListFunctions"/>
+        <property name="uri.var.maxItems" expression="$func:maxItems"/>
+        <property name="httpMethod" value="GET"/>
+        <property name="uri.var.uriRemainder"
+                  expression="fn:concat('/', $ctx:uri.var.apiVersion, '/functions/')"/>
+        <script language="js">
+            <![CDATA[
+                var queryParameters = '';
+                var functionVersion = mc.getProperty('uri.var.functionVersion');
+                var marker = mc.getProperty('uri.var.marker');
+                var masterRegion = mc.getProperty('uri.var.masterRegion');
+                var maxItems = mc.getProperty('uri.var.maxItems');
+                if (functionVersion != null && functionVersion != "")
+                {
```

Comment:
Move this to above line . Fix the same issue in all the places

Reply:
Fix with d1edbe1

Reply:
Fix with 408c7bd